tensorflow_addons/text/BUILD

```diff
@@ -6,6 +6,7 @@ py_library(
     name = "text",
     srcs = ([
         "__init__.py",
+        "crf_ops.py",
```

Comment:
I think we can just name this as `crf` and `crf_tests`. For reference `skip_gram_ops`, `distort_image_ops`, etc. all utilize custom_ops